src/polyhedron.jl

```diff
@@ -55,8 +55,6 @@ function getine(p::Polyhedron)
         else
             p.ine = LiftedHRepresentation(getextm(p, :Fresh))
             p.inem = nothing
-            p.hlinearitydetected = true
```

Comment:
Leave the setting of `hlinearitydetected` here and add a mention to the issue to say that in some examples there are still redundant elements

Reply:
Changed back `p.hlinearitydetected = true` and `p.vlinearitydetected = true` in `getine` and `getext`.

(By the way, please feel free to edit this branch.)